var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -45,11 +45,13 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     # URL must remain http:// so Spack can bootstrap curl
     url = "http://www.cpan.org/src/5.0/perl-5.24.1.tar.gz"
 
+    version('5.26.2', 'dc0fea097f3992a8cd53f8ac0810d523')
+
     # Development releases
     version('5.25.11', '37a398682c36cd85992b34b5c1c25dc1')
 
     # Maintenance releases (recommended)
-    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c', preferred=True)
+    version('5.24.1', '765ef511b5b87a164e2531403ee16b3c')
     version('5.22.3', 'aa4f236dc2fc6f88b871436b8d0fda95')
```

Comment:
Can you move 5.22.3 to the "Misc releases" section? Can you also add 5.22.4 to the "End of life" section?

Reply:
5.22.4? we don't have it in spack.